classes/statistics/TemporaryInstitutionsDAO.php

```diff
@@ -18,8 +18,9 @@
 
 use Illuminate\Support\Collection;
 use Illuminate\Support\Facades\DB;
+use PKP\db\DAO;
 
-class TemporaryInstitutionsDAO
+class TemporaryInstitutionsDAO extends DAO
```

Comment:
@bozana, could you review this commit?

Reply:
Hmmm... All those DAOs in classes/statistics/ do actually not need to extend DAO -- are a little bit different than other objects DAOs -- but I suppose there is no harm this way... :thinking:

Reply:
My reasoning here was the following: If all DAOs are extending this base class, so I think it's better to keep the pattern, it's not going to harm, and might be useful somewhere (e.g. `$dao instanceof DAO` ? a : b`).

In another PR I was prototyping something that would require such check 🤔

Reply:
I have no problem with extending the DAOs as a general principle, but we don't have a pressing need to do it for 3.4.0 and I'd rather not make speculative changes just before the release. Can we leave this for `main` or at least `stable-3_4_0` post 3.4.0-0?

Reply:
Yep, this is definitely not a requirement.